### PR TITLE
My Projects pt 3: Basic Structure and Routing

### PR DESCRIPTION
**Pull request by rajadain**
## Overview

Builds on top of #903 and translates the mock-up into a basic set of views, to look like this:

image

Summary of changes:

 * 44fd081 Switch to a light-weight project serializer which gives us only the information we need for the "My Projects" screen
 * c1fd1fc Has the actual view breakdown, and does the following:
   - Adds a `ProjectCollection`
   - Decomposes mockup into separate templates and adjusts styling
   - Adds a new `toFriendlyDate` Nunjucks filter for date format assist
   - The projects are populated and show the `name`, `area_of_interest_name`, `created_at` fields, the lock / globe icon is shown based on `is_private`, and the "Launch Project" button points to the correct project URL based on the project's `id`
 * d5d92c1 Adds navigation to the "My Projects" page from the user dropdown in the header and the project menu for logged in users.

Still remaining:

 * Search functionality
 * "Create New Project" button
 * Rename, Share, Make Public / Private, Delete functionality

Known bugs:

 * User dropdown has stopped working in Model view

Connects #820

### CODE REVIEW

File: src/mmw/js/src/projects/templates/projectRow.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<div class="col-md-5">
+    <h2><i class="fa {{ "fa-lock" if is_private else "fa-globe" }}"></i> {{  name }}</h2>
+    <p>{{ area_of_interest_name }}</p>
+    <p><small>{{ created_at|toFriendlyDate }}</small></p>
+</div>
+<div class="col-md-offset-1 col-md-4">
+    <div class="btn-group">
+        <button class="btn btn-default">Rename</button>
+        <button class="btn btn-default">Share</button>
+        <button class="btn btn-default">Make {{ "Public" if is_private else "Private" }}</button>git c
```
Comment (lliss) [inline src/mmw/js/src/projects/templates/projectRow.html]: Looks like some accidental typing... `git c`

Comment (rajadain) [inline src/mmw/js/src/projects/templates/projectRow.html]: urgh good catch

**Comment by rajadain** [comment]
The failing tests are because of the addition of the "My Projects" link in the Project Dropdown. Will fix shortly.

**Comment by lliss** [comment]
+1. Looks solid all around.

**Comment by mmcfarland** [comment]
Yeah, this is looking really good.

**Comment by rajadain** [comment]
Fixed typos and failing tests. Moving on to implement the rest ...

**Comment by rajadain** [comment]
Updated to not let `itsi_embed` users see the "My Projects" screen

**Comment by rajadain** [comment]
Thanks for reviewing all! Closing this in favor of #908

**Pull request closed without merging**